### ISSUE TITLE
Fix pyflakes for spread

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6989]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#6989|
|Type|enhancement|
|Created|2014-02-27 12:02:41Z|
|Branch|https://github.com/twisted/twisted/tree/pyflakes-spread-6989|

I have explicitly defined the _PREFIX_LIMIT and explicitly import from Tkinter.

Attachments:

* [6989-1.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a17/a17c3e6ca0d2c34f8db9cb4d59a0733e888986a7/78603d3ae6bbd75892ffafbf1f3a2fcebf986298.diff) (1476 bytes) - added by adiroiban on 2014-02-27 12:11:23Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6989 6989
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__ 
branch__branches_pyflakes_spread_6989 branches/pyflakes-spread-6989
branch_author__rwall rwall
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1393502561000000 1393502561000000
changetime__1397458734000000 1397458734000000
version__None None
owner__rwall rwall

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)|@wallrj set owner to @wallrj<br>@wallrj set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_3"></a>|@wallrj commented|
|-|-|

(In [42184]) Branching to 'pyflakes-spread-6989'

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_4"></a>|@wallrj commented|
|-|-|

(In [42185]) Apply 6989-1.diff by adiroban. Refs [#6989](https://github.com/twisted/twisted/issues/6989).

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_5"></a>|@wallrj set status to `new`|
|-|-|

Thanks Adi,

Notes:
 * No pyflakes errors in spread after applying the patch.
 * Tests pass locally after applying the patch.
 * Started a build: https://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/pyflakes-spread-6989

Points:
 1. Maybe the imported Tkinter modules could be listed inline to save vertical space.

Otherwise this looks good. I'll merge if all the builds pass.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_6"></a>|@wallrj set status to `closed`|
|-|-|

(In [42201]) Merge pyflakes-spread-6989

Author: adiroiban
Reviewer: rwall
Fixes: [#6989](https://github.com/twisted/twisted/issues/6989)

Cleanup unused imports and uninitialised variables in twisted.spread. Twisted
Spread is now "pyflakes clean".
